index.html

```diff
@@ -304,40 +304,42 @@ <h2>Contact me</h2>
             method="post"
             class="form-group"
           >
-          <div>
-            <input
-            type="text"
-            name="name"
-            class="form-control"
-            id="name"
-            size="30"
-            placeholder="Joseph"
-            required
-            />
-          </div>
-          <div>
-            <input
-            type="email"
-            name="email"
-            class="form-control"
-            id="email"
-            placeholder="drack.sir01@gmail.com"
-            required
-            />
-          </div>
-          <div>
-            <textarea
-            name="message"
-            class="form-control text-area"
-            id="message"
-            cols="30"
-            rows="10"
-            maxlength="500"
-            placeholder="Write your message here"
-            required
-            ></textarea>
-          </div>
-            <button class="contact-btn" type="submit">Get In Touch</button>
+            <div>
+              <input
+                type="text"
+                name="name"
+                class="form-control"
+                id="name"
+                size="30"
```

Comment:
- [ ] Kindly change the `size` for input text to `max length` as per requirements.